install/update.php

```diff
@@ -840,7 +848,7 @@ function updateDbUpTo031() {
    }
 
    // Update version number and default langage and new version_founded ---- LEAVE AT THE END
-   Config::setConfigurationValues('core', array('version'             => GLPI_VERSION,
+   Config::setConfigurationValues('core', array('version'             => GLPI_SCHEMA_VERSION,
```

Comment:
It may be a solution to fix the very first migration when merged. Will take a look, thank you for the idea.